crates/numeric_cast/src/truncating.rs

```diff
@@ -51,6 +51,18 @@ truncating_cast!(u128 => u16 );
 truncating_cast!(u64  => u32 );
 truncating_cast!(u128 => u32 );
 truncating_cast!(u128 => u64 );
+
+truncating_cast!(usize => u8    );
+truncating_cast!(usize => u16   );
+truncating_cast!(usize => u32   );
+truncating_cast!(usize => u64   );
+truncating_cast!(usize => u128  );
+truncating_cast!(u8    => usize );
```

Comment:
`u8/i8 => usize/isize` is extending cast since `usize/isize` are at least 16-bits wide.